Dockerfile.base

```diff
@@ -116,13 +108,17 @@ FROM base as ci-wpcom
 COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
 COPY --from=cache --chown=$UID /calypso/composer.* /calypso/
 
-RUN apt update &&\
-	apt-get install -y apt-transport-https &&\
-	wget https://packages.sury.org/php/apt.gpg -O /etc/apt/trusted.gpg.d/php-sury.gpg &&\
-	echo "deb https://packages.sury.org/php/ buster main" > /etc/apt/sources.list.d/php-sury.list &&\
-	apt update &&\
- 	apt-get upgrade -y &&\
-	apt-get install -y php7.4-cli php7.4-xml php7.4-mbstring docker-compose &&\
+RUN apt update && \
+	apt install -y apt-transport-https wget
```

Comment:
I had to separate the two calls here, otherwise Docker build would throw an error that `wget` is not available.